xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
Ah! sorry... That makes more sense! I'd have to check whether Spring whitelists its NS'es for that... no idea. That being said, I still think that consistency is desirable (again, as this is all targets configuring the _same_ thing, i.e. a `CacheManager`)